L1Trigger/TrackerTFP/test/AnalyzerDR.cc

```diff
@@ -32,16 +31,16 @@ using namespace edm;
 using namespace trackerTFP;
 using namespace tt;
 
-namespace trklet {
+namespace trackerTFP {
 
-  /*! \class  trklet::AnalyzerKFin
-   *  \brief  Class to analyze hardware like structured TTStub Collection generated by Tracklet
+  /*! \class  trackerTFP::AnalyzerDR
+   *  \brief  Class to analyze hardware like structured TTStub Collection generated by Duplicate Removal
```

Comment:
Since this consumes StreamsStub and StreamsTrack, is it accurate to say it's analyzing TTStubs? And logically, it must be looking at tracks, to check DR performance.

Reply:
done